drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -6239,12 +6246,12 @@ void RasterizerStorageGLES2::initialize() {
 	config.s3tc_supported = config.extensions.has("GL_EXT_texture_compression_s3tc") || config.extensions.has("WEBGL_compressed_texture_s3tc");
 	config.etc1_supported = config.extensions.has("GL_OES_compressed_ETC1_RGB8_texture") || config.extensions.has("WEBGL_compressed_texture_etc1");
 	config.pvrtc_supported = config.extensions.has("GL_IMG_texture_compression_pvrtc") || config.extensions.has("WEBGL_compressed_texture_pvrtc");
-	config.support_npot_repeat_mipmap = config.extensions.has("GL_OES_texture_npot");
+	config.support_npot_repeat_mipmap = config.extensions.has("GL_OES_texture_npot") || config.extensions.has("GL_IMG_texture_npot");
 
 	// If the desktop build is using S3TC, and you export / run from the IDE for android, if the device supports
 	// S3TC it will crash trying to load these textures, as they are not exported in the APK. This is a simple way
 	// to prevent Android devices trying to load S3TC, by faking lack of hardware support.
-#if defined(ANDROID_ENABLED) || defined(IPHONE_ENABLED)
+#if defined(ANDROID_ENABLED) || defined(IPHONE_ENABLED) || defined(VITA_ENABLED)
```

Comment:
This is happening in two different places for vita/switch.

Reply:
- [ ] Add `|| HORIZON_ENABLED` to this block